tests/http/__init__.py

```diff
@@ -145,7 +145,7 @@ def __init__(self, sanlist):
         self._cert_file = create_test_cert_file(sanlist)
 
     def serverConnectionForTLS(self, tlsProtocol):
-        ctx = SSL.Context(SSL.TLSv1_METHOD)
+        ctx = SSL.Context(SSL.TLSv1_2_METHOD)
```

Comment:
as above: please can you use the right constant here?

Reply:
Sure. I checked that `SSLv23_METHOD` also fixes the test failures with `MinProtocol = TLSv1.2`. I've pushed an extra to commit to change it to the `SSLv23_METHOD` constant.